yarn.lock

```diff
@@ -2157,6 +2207,17 @@
     "@swc/helpers" "^0.5.0"
     clsx "^2.0.0"
 
+"@react-aria/focus@^3.16.2":
```

Comment:
seeing lots of duplicates

Reply:
true! any recommended tool we normally use for de-duping?

Reply:
Used `yarn-deduplicate`